src/pages/profile/ProfileSections/ProfileAllPosts.jsx

```diff
@@ -0,0 +1,20 @@
+import RecipeCard from "../../../components/TypesOfRecipeCards/RecipeCard";
+import ReviewCard from "../../../components/TypesOfRecipeCards/ReviewCard";
+const ProfileAllPosts = ({ recipes, reviews }) => {
+  //TODO
+  //Sort by date
+
+  return (
+    <div className="profile-all-posts mt-4 grid grid-cols-3">
+      {[...recipes, ...reviews].map((item, i) =>
+        "recipeId" in item ? (
```

Comment:
I do not like this way of determining whether it's a recipe. Very confusing. Can you think of another way?